app/src/main/java/com/denisatrif/truthdare/viewmodel/PlayersViewModel.kt

```diff
@@ -43,19 +41,35 @@ class PlayersViewModel @Inject constructor(private val playersRepository: Player
         return liveData
     }
 
-    fun startGame() {
-        addAllPlayers()
-    }
-
-    private fun deleteAll() {
+    fun deletePlayer(player: Player) {
+        _playersList.remove(player)
         viewModelScope.launch(Dispatchers.IO) {
-            playersRepository.deleteAll()
+            playersRepository.delete(player)
         }
     }
 
-    fun deletePlayer(player: Player) {
+    fun getAllIds(): Flow<List<Int>> {
+        return playersRepository.getListOfIds()
+    }
+
+    fun getNext(id: Int): Flow<Player> {
+        return playersRepository.getPlayerWithId(id)
+    }
+
+    private fun getCount(): LiveData<Int> {
```

Comment:
Can we do this using Flows? just to get rid of all LiveData references

Reply:
Done